api/lib/infrastructure/serializers/jsonapi/badge-with-learning-content-serializer.js

```diff
@@ -19,17 +20,37 @@ module.exports = {
         'title',
         'isCertifiable',
         'badgeCriteria',
+        'skillSets',
         'badgePartnerCompetences',
       ],
       badgeCriteria: {
         include: true,
         ref: 'id',
-        attributes: ['threshold', 'scope', 'partnerCompetences'],
+        attributes: ['threshold', 'scope', 'skillSets', 'partnerCompetences'],
```

Comment:
```suggestion
        attributes: ['threshold', 'scope', 'skillSets'],
```
On en a plus besoin normalement

Reply:
@MelanieMEB on garde `partnerCompetences` pour s'assurer que la version du front qui utilise encore `partnerCompetences` fonctionne (le front met plus de temps à se déployer que l'api)